docs/manual/manual_redundant_throttle.rst

```diff
@@ -0,0 +1,112 @@
+source if the active source goes offline. The second set of parameters defines the priorities of each supported throttle source. You can find the specific parameters 
+at :ref:`throttle_source_manager`.
+
+Vertiq's supported throttle sources are :ref:`DroneCAN <dronecan_protocol>`, :ref:`IQUART <uart_messaging>`, and :ref:`hobby protocols <hobby_protocol>`. Please note that only priority values corresponding to protocols supported by 
+your module have any effect. See your module's family page to see what throttle sources are supported.
+
+Priorities can be valued from 0 to 3. Setting a priority of 0 indicates that the module will ignore all throttle messages received from the configured source. 
+Priority values [1, 3] define the priority of each protocol against the others where 3 defines the highest priority.
+
+.. note:: 
+    Due to Vertiq modules' hardware, it is not possible to use both IQUART and hobby protocols simultaneously. As such, it is only possible to use DroneCAN with one of IQUART 
+    or hobby as redundant throttle sources.
+
+Configuration Example 1
+##########################
+
+Suppose your module supports DroneCAN, IQUART, and hobby inputs. 
+
+You configure the following:
+
+- IQUART's priority to 3 and DroneCAN's to 1 (the module will automatically set hobby's to 2 ensuring that there are no two matching priorities)
+- Flight controller outputting both :ref:`IQUART Flight Controller Interface commands <controlling_ifci>` as well as DroneCAN commands
+- Throttle source timeout configured to 0.5 seconds
+
+The sequence diagram below illustrates how your module reacts to received throttle commands as well as how it deals with switching between sources.
+
+.. image:: ../_static/manual_images/redundant_throttle/iquart_prio_example.png
+    :width: 50%
+
+Note that the first DroneCAN throttle is applied to the motor since no IQUART messages had been received before. Then, when both an IQUART and DroneCAN message are received, 
+the IQUART throttle is applied. Last, DroneCAN throttles are only applied again once the throttle timeout is reached after IQUART disconnects.
+
+Configuration Example 2
+##########################
+
+Suppose your module supports DroneCAN, IQUART, and hobby inputs. 
+
+You configure the following:
+
+- DroneCAN's priority to 3, IQUART to 0, hobby to 0
+- Flight controller outputting both :ref:`hobby_dshot` as well as DroneCAN commands
+- Throttle source timeout configured to 0.1s
+
+.. image:: ../_static/manual_images/redundant_throttle/dshot_ignored_ex.png
+    :width: 70%
+
+In this example, the module is configured to ignore all hobby and IQUART throttle messages. So, all received DSHOT throttles are dropped, and throttles are only applied once DroneCAN 
+throttles are received. An important note from this example is that the module will reach the module's :ref:`propeller motor control timeout <manual_timeout>` even if the 

```

Comment:
Maybe rephrase this sentence to: "An important note from this example is that the module will reach **its** propeller motor control timeout even if the flight controller continues to send DSHOT throttles.